Description: localize $_ before reading over <DATA> as $_ may be aliased
 For example, this would fail: for ( 1, 2 ) { require Net::LDAP::Constant }
 This posed a problem for lemonldap-ng's test suite
Bug-Debian: 577340
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=56726
Author: Damyan Ivanov <dmn@debian.org>
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2010-04-19

--- a/lib/Net/LDAP/Constant.pm
+++ b/lib/Net/LDAP/Constant.pm
@@ -10,6 +10,7 @@ use Exporter qw(import);
 
 my @err2name;
 
+local $_;
 while(<DATA>) {
   last if /^=cut/;
   my $protocol_const = /^=head2 Protocol Constants/ ... /^=head2/;
